lib/cmd/cov.js

```diff
@@ -25,6 +25,10 @@ class CovCommand extends Command {
         description: 'prerequire files for coverage instrument',
         type: 'boolean',
       },
+      nyc: {
+        description: 'nyc instruments passthrough',
+        type: 'string',
```

Comment:
how about move `covArgs` to this default value ?

Reply:
done